CHANGELOG.md

```
@@ -1,5 +1,41 @@
+## [1.1.22] - 2025-07-26
+
+**Chores**
+- 워크플로우가 \"CHANGELOG 자동 업데이트\" 완료 시 자동으로 실행되도록 여러 배포 및 파일 업로드 작업의 트리거가 추가되었습니다.
+- pubspec.yaml의 버전이 1.1.21+1.1.0으로 업데이트되었습니다.
+- 워크플로우 내 일부 안내 문구가 더 간결하게 수정되었습니다.
+
+---
+
+## [1.1.21] - 2025-07-26
+
+**Chores**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)